samcli/commands/check/resources/Graph.py

```diff
@@ -11,6 +13,7 @@ class Graph:
     yellow_warnings: List
     red_warnings: List
     red_burst_warnings: List
+    lambda_function_pricing_info: LambdaFunctionPricing
 
     def __init__(self):
         self.entry_points: List = []
```

Comment:
can you be more specific while using Typing. as an example, this variable if of type List, can you be more specific and mention list of what